### ISSUE TITLE
Unknown config attribute with test_tsv profile with Nextflow 20.04

### ISSUE THREAD

**Issue by maxibor** [nf-core/eager#631]
## Description of the bug

Running the `dev` branch of eager, (as of commit b916b6d) with the `test_tsv` profile and Nextflow version 20.04.1 leads to this error:  

```bash
$ nextflow run main.nf -profile test_tsv,singularity --single_end
N E X T F L O W  ~  version 20.04.1
Launching `main.nf` [nasty_coulomb] - revision: 0cb20d25f3
Unknown config attribute `params.genomes.GRCh37.projectDir` -- check config file: /projects1/users/borry/23_eager/test/eager/nextflow.config
```

Running it NF 20.10 fixes the issue.

## Steps to reproduce

Steps to reproduce the behaviour:

```
$ nextflow run main.nf -profile test_tsv,singularity --single_end
```

- Hardware: Slurm HPC, and Mac

## Nextflow Installation

- Version: 20.04.1 -->

## Container engine

- Singularity and Docker

**Comment by jfy133** [comment]
```
$ export NXF_VER=20.04.0
$ nextflow run main.nf -profile test_tsv,singularity --single_end
N E X T F L O W  ~  version 20.04.0
Launching `main.nf` [reverent_feynman] - revision: fe08e6a5fa
----------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/eager v2.2.3dev
----------------------------------------------------
$ export NXF_VER=20.04.1
$ nextflow run main.nf -profile test_tsv,singularity --single_end
N E X T F L O W  ~  version 20.04.1
Launching `main.nf` [clever_banach] - revision: fe08e6a5fa
Unknown config attribute `params.genomes.GRCh37.projectDir` -- check config file: /home/jfellows/Documents/git/maxibor/eager/nextflow.config
$ export NXF_VER=20.05.0-edge
$ nextflow run main.nf -profile test_tsv,singularity --single_end
N E X T F L O W  ~  version 20.05.0-edge
Launching `main.nf` [fabulous_booth] - revision: fe08e6a5fa
----------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/eager v2.2.3dev
----------------------------------------------------
```

This was a specific bug in Nextflow 20.04.1, which was then fixed in the next release. This is an edge case so we can close for now

**Comment by jfy133** [comment]
We need to bump the minimum Nextflow version for eager unfortunately. By default e.g. aws will take the latest patch release of a version, so even though `20.04.0` does work, AWS will pull `20.04.1` and will crash :\

**Comment by apeltzer** [comment]
Fixed by #663
